Errors on generating Javadoc with Maven and AdoptOpenJDK

I tried to build the code (mvn package) so I could use the "all-in-one" JAR file to run the tests locally but I got the following errors:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:03 min
[INFO] Finished at: 2020-03-26T21:50:48+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.7.1:jar (site-package) on project ets-ogcapi-features10: Error generating maven-javadoc-plugin:2.10.4:javadoc report:
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR] C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\src\main\java\org\opengis\cite\ogcapifeatures10\util\JsonUtils.java:215: warning: no @param for rels
[ERROR]     public static List<String> findLinksWithoutRelOrType( List<Map<String, Object>> links, Set<String> rels ) {
[ERROR]                                ^
[ERROR]
[ERROR] Command line was: C:\OtherPrograms\AdoptOpenJDK\x64_11.0.2_9\jdk-11.0.2+9\bin\javadoc.exe @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in 'C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\target\site\apidocs' dir.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Updating the javadoc for method `JsonUtils.findLinksWithoutRelOrType()` resolves one issue, but the build still fails:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:46 min
[INFO] Finished at: 2020-03-26T21:55:32+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project ets-ogcapi-features10: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR]
[ERROR] Command line was: C:\OtherPrograms\AdoptOpenJDK\x64_11.0.2_9\jdk-11.0.2+9\bin\javadoc.exe @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in 'C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\target\apidocs' dir.
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/pom.xml
+++ b/pom.xml
@@ -109,6 +109,7 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <version>2.10.4</version>
         <configuration>
+          <source>1.8</source>
           <docfilessubdirs>true</docfilessubdirs>
           <show>package</show>
           <links>

